VMBackup/main/handle.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+##/var/log/azure/Microsoft.Azure.RecoveryServices.VMSnapshotLinux/
+pwdcommand=`pwd`
+pwdstr="$pwdcommand"
+logfile='/var/log/azure/Microsoft.Azure.RecoveryServices.VMSnapshotLinux/'${pwdstr#*-}'/shell.log'
```

Comment:
Get this path from HandlerEnvironment.json

Reply:
done